environments/mcdTransformers.toml

```diff
@@ -1,1849 +1,1175 @@
-        deployed = 10144450
-    [contract.MCD_FLIP_USDC_B_1_0_9]
-        address  = "0x6002d3B769D64A9909b0B26fC00361091786fe48"
-        abi      = '[{"inputs":[{"internalType":"address","name":"vat_","type":"address"},{"internalType":"bytes32","name":"ilk_","type":"bytes32"}],"payable":false,"stateMutability":"nonpayable","type":"constructor"},{"anonymous":false,"inputs":[{"indexed":false,"internalType":"uint256","name":"id","type":"uint256"},{"indexed":false,"internalType":"uint256","name":"lot","type":"uint256"},{"indexed":false,"internalType":"uint256","name":"bid","type":"uint256"},{"indexed":false,"internalType":"uint256","name":"tab","type":"uint256"},{"indexed":true,"internalType":"address","name":"usr","type":"address"},{"indexed":true,"internalType":"address","name":"gal","type":"address"}],"name":"Kick","type":"event"},{"anonymous":true,"inputs":[{"indexed":true,"internalType":"bytes4","name":"sig","type":"bytes4"},{"indexed":true,"internalType":"address","name":"usr","type":"address"},{"indexed":true,"internalType":"bytes32","name":"arg1","type":"bytes32"},{"indexed":true,"internalType":"bytes32","name":"arg2","type":"bytes32"},{"indexed":false,"internalType":"bytes","name":"data","type":"bytes"}],"name":
```

Comment:
It looks like "FLIP_BAL_A" was passed in as the collateral name, instead of just "BAL_A",  which is why there is a duplicate "FLIP" here. The generator should take care of prefixing the collateral name with the appropriate contract type ("FLIP", "OSM" or "MEDIAN") - so this should just be "MCD_FLIP_BAL_A_1_1_4". 

In other words, I think we'll need to rerun the generator with the collateral name as "BAL_A" passed in.